examples/text_labels_and_annotations/dfrac_demo.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python2
```

Comment:
Will do. This is my first contribution, so I'm still working out some of the bugs. Thanks for the prompt suggestions.